### PR TITLE
chore: publish v1.6.0 with experimental signInWithIdToken for Apple, Google and more http methods support on functions

### PR DESCRIPTION
**Pull request by DanMossa**
## What kind of change does this PR introduce?

This PR adds support to use the new `signInWithIdToken` introduced in gotrue v1.5.0

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,7 +9,11 @@ environment:
 
 dependencies:
   functions_client: ^1.0.2
-  gotrue: ^1.4.2
```
Comment (DanMossa) [inline pubspec.yaml:12]: Waiting for https://github.com/supabase/gotrue-dart/pull/120

**Comment by Vinzent03** [comment]
Nothing specific to the changes in gotrue, but I don't think we need to bump the version in supabsae-dart/flutter for every release in a sub library. Users should check for outdated transitive dependencies as well (`dart pub outdated`).

File: pubspec.yaml (hunk 1 above)
Comment (dshukertjr) [reply pubspec.yaml:12]: Just released a new version!

**Comment by dshukertjr** [comment]
If we could include this PR into the release as well, it would be great!
https://github.com/supabase/supabase-dart/pull/179

**Comment by dshukertjr** [comment]
I think the [minor version](https://semver.org/) should be bumped since that is the update on Gotrue. Let me quickly fix that!

**Comment by dshukertjr** [comment]
I think it's all good now! If someone could review this one https://github.com/supabase/supabase-dart/pull/179 so that we could also include it in the release, that would be awesome!

**Comment by DanMossa** [comment]
@dshukertjr Any updates on when we feel like merging this?

**Review by dshukertjr — APPROVED** [review]
Sorry for that! It slipped by me. Will publish this version right now!

**Pull request merged**